packages/vue/src/components/molecules/SfPagination/SfPagination.vue

```diff
@@ -202,10 +202,9 @@ export default {
       this.$emit("click", page);
     },
     getLinkTo(page) {
-      return {
-        ...this.$route,
-        query: { ...this.$route.query, [this.pageParamName]: page },
-      };
+      Object.assign(this.$route.query, { [this.pageParamName]: page });
```

Comment:
Do we want to modify `this.$route.query` or we just want to get the link to page from Object to string?

If we just want to get the link to page, we don't need this line.
If we do, we need to make sure there is no effect when we change `$route.query`
@justyna-13 please check

Reply:
right, we don't need to modify it, works without it :)